frontend/src/app/forms/forms.page.ts

```diff
@@ -48,7 +55,7 @@ export class FormsPage implements OnInit, OnDestroy {
     isActive: boolean;
     selector: string;
   };
-
+  buttonText: string;
```

Comment:
This property was not used any where now. I suggest to use this property to show continue or submit instead of calling methods in template.